borg/archiver.py

```diff
@@ -60,10 +59,9 @@ def print_warning(self, msg, *args):
         self.exit_code = EXIT_WARNING  # we do not terminate here, so it is a warning
         logger.warning(msg)
 
-    def print_verbose(self, msg, *args):
-        if self.verbose:
-            msg = args and msg % args or msg
-            logger.info(msg)
+    def print_file_status(self, status, path):
+        if self.args.changed:
+            print("%1s %s" % (status, remove_surrogates(path)), file=sys.stderr)
```

Comment:
i just noticed an issue with --changed. as in your cs, it outputs all files (no matter what status)
so, you'ld have to check whether it is A or M.
and only output all the stuff for --no-changed (or how you'ld call the other thing)
and output nothing, if none of both flags is given